AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -22,6 +22,11 @@ @interface MSAuthTokenContext ()
  */
 @property(nonatomic) NSHashTable<id<MSAuthTokenContextDelegate>> *delegates;
 
+/**
+ * YES if the current token should be reset.
+ */
+@property(atomic) BOOL resetAuthTokenRequired;
```

Comment:
nit: `atomic` is default so you don't it unless you _really_ want to emphasize it.